connect_transformations/static/transformations/formula.fe204b5e645d87ba99d5.js

```diff
@@ -646,10 +646,15 @@ const formula = (app) => {
     } = config;
 
     columns = availableColumns;
-    suggestor = { '.': availableColumns.map(col => ({
-      title: col.name,
-      value: `."${col.name}"`,
-    })) };
+    const pattern = /[ .,|-]/;
```

Comment:
I would suggest to wrap with quotes also if there is one of:

`{}[]+/*%:;`

Reply:
good idea, fixed!